samples/manage/azure-automation-automated-export/README.md

```diff
@@ -0,0 +1,53 @@
+---
+services: azure automation
+platforms: azure
+author: trgrie-msft
+---
+
+# Setting up Auto Export in Azure Automation
+
+Provides the scripts and lists the steps to set up automatically exporting your databases to Azure Storage with Azure Automation.
+
+## Azure Automation Set Up
+
+1. Create and uploade the certificates that you will use to authenticate your connection to azure.
+	- Run powershell as admin.
+	- Run the New-SelfSignedCertificate command: New-SelfSignedCertificate -CertStoreLocation cert:\localmachine\my -DnsName <certificateName>
+	- Create a corresponding pfx certificate by taking the thumbprint of the newly created certificate and running these commands:
+		- $CertPassword = ConvertTo-SecureString -String <YourPassword> -Force -AsPlainText
+		- Export-PfxCertificate -Cert cert:\localmachine\my\<thumbprint> -FilePath <PathAndFileName>.pfx -Password $CertPassword
+	- Upload the .cer file to your subscription [here][https://manage.windowsazure.com/]
+	- Upload the .pfx file to the certificates under Assets in the automation account that you want to use on Azure. You will use the password you gave in the previous step to authenticate it.
+2. Create new a new credentials asset to authenticate your server with.
```

Comment:
There are 2 "new" in this sentence. Also, better prefix "server" with "Azure SQL database" server to remove confusion.